### DISCUSSION TITLE
Bug: Git-App freeze and redirect to repo by viewing particular commit.

### DISCUSSION THREAD

**Discussion by THElegend5** [community/community#57612] [Mobile] [answered]
If you tap on a commit in the commits tab to view that particular commit instead of taking you to that commit github freezes with a black screen for a second and take you back to the repo's home page.

App version : 1.117.0
Android version: 13
Device: Samsung S21 FE

**Comment by DSDmark** [comment]
Hi @THElegend5,

if it's a bug. Then, add **bug** in this discussion labels. And write title like:

> Bug: Git-App freeze and redirect to repo by viewing particular commit.

And write some more details about that bug. Also add which device and OS you are facing the bug.

It helps to understand other member of the community, and it catches the attention of development team.

**Comment by iskae** [comment, marked as answer]
Hi @THElegend5 👋 

Thanks so much for reaching out ❤️ We were able to reproduce this issue and released a fix as soon as possible 🎉 
Can you please check out the latest version from this week and let us know if it fixed this issue? 🙇 

Thanks in advance 😊

**Reply by THElegend5** [reply]
Yep the issue is fixed.
